### ISSUE TITLE
Turris MOX (aarch64) packages

### ISSUE THREAD

**Issue by Kedlub** [mato/tailscale-turris#3]
This package only supports the 32-bit ARM architecture used by older models like the Turris Omnia. However, Turris MOX routers use the 64-bit aarch64 architecture, and therefore this package doesn't work on MOX devices. Would it be possible to also build and publish this package for the aarch64 architecture?

**Comment by mato** [comment]
It's certainly possible, it involves some amount of work and I'd need
either remote access to a Turris MOX device and/or someone with a device
to hang out with on IRC/chat to test the result.

Would you be willing to help out with this?

**Comment by ArzykDev** [comment]
Yes, I would like to help out. We will probably have to use chat, as I don't have a public IP for remote access.
